dtfs-central-api/src/services/state-machines/utilisation-report/event-handlers/helpers/calculate-initial-utilisation-and-fixed-fee.test.ts

```diff
@@ -109,34 +112,42 @@ describe('helpers/calculate-initial-utilisation-and-fixed-fee', () => {
   });
 
   describe('calculateInitialUtilisationAndFixedFee', () => {
-    const findOneByUkefFacilityIdSpy = jest.spyOn(TfmFacilitiesRepo, 'findOneByUkefFacilityId');
+    const getKeyingSheetCalculationTfmFacilityValuesSpy = jest.spyOn(helpers, 'getKeyingSheetCalculationTfmFacilityValues');
     const facilityId = '12345678';
+    const { facilitySnapshot } = aTfmFacility();
+
+    const latestTfmFacilityValues = {
```

Comment:
Please can we rename this since this wording is no longer relevant

Reply:
changed